AS flaky test

<!--
Thanks for submitting a bug report. Please fill the template below,
otherwise we will not be able to process this bug report.
-->

#### Summary
<!-- Summarize the problem in a few sentences: -->

AS flaky test

#### Steps to Reproduce
<!-- How can we reproduce the problem? -->

Happens sometimes in CI

<!--
Please upload relevant configuration (as .txt).
If you use the command "ttn-lw-stack config", you can redact sensitive config.
-->

#### What do you see now?
<!--
Please paste terminal output, upload logs (as .txt) or upload screenshots.
-->

```
  WARN No cookie hash key configured, generated a random one hash_key=C0BEFF6FA83E3334E8C53405A2D2CBF109822059E150FED74905FBA698966F7619572DE8016D8212580ED431EA20BD6D0CAFA2910268962A735BCE16640F896E namespace=web test_name=TestApplicationServer
  WARN No cookie block key configured, generated a random one block_key=38E78EFE86E4D4AD2B3A0ED3F59CF9A203FF77DA50F826D7D1A98672B554F76B namespace=web test_name=TestApplicationServer
  WARN No static assets found in any search path namespace=web search_paths=[] test_name=TestApplicationServer
  WARN No rights TTL configured                 test_name=TestApplicationServer
  WARN No tenant cache TTL configured           test_name=TestApplicationServer
  WARN No cluster key configured, generated a random one key=52b435459d66dc26ccb0e5bd9ae754daef4ef523977712763ec6b729a7862dba test_name=TestApplicationServer
  INFO Listening for connections                address=:9184 namespace=grpc protocol=gRPC test_name=TestApplicationServer
  INFO Listening for connections                address=:8099 namespace=web protocol=Web test_name=TestApplicationServer
  INFO Linked                                   application_uid=foo-app@foo-tenant namespace=applicationserver network_server=ns tenant_id=foo-tenant test_name=TestApplicationServer
  INFO Subscribed                               application_uid=foo-app@foo-tenant grpc_method=Subscribe grpc_service=ttn.lorawan.v3.AppAs namespace=applicationserver/io/grpc remote_addr=pipe request_id=01DPS7QGKQ7G10TRF9D549P865 test_name=TestApplicationServer
  INFO Unsubscribed                             application_uid=foo-app@foo-tenant grpc_method=Subscribe grpc_service=ttn.lorawan.v3.AppAs namespace=applicationserver/io/grpc remote_addr=pipe request_id=01DPS7QGKQ7G10TRF9D549P865 test_name=TestApplicationServer
  WARN Finished streaming call                  duration=511.951347ms error=context deadline exceeded grpc_method=Subscribe grpc_service=ttn.lorawan.v3.AppAs namespace=grpc request_id=01DPS7QGKQ7G10TRF9D549P865 test_name=TestApplicationServer
  INFO Unlinked                                 application_uid=foo-app@foo-tenant namespace=applicationserver network_server=ns tenant_id=foo-tenant test_name=TestApplicationServer
  WARN transport: http2Server.HandleStreams failed to read frame: io: read/write on closed pipe namespace=grpc test_name=TestApplicationServer
--- FAIL: TestApplicationServer (2.15s)
    --- FAIL: TestApplicationServer/Authenticate/grpc (0.51s)
        --- FAIL: TestApplicationServer/Authenticate/grpc/ValidIDAndKey (0.51s)
            applicationserver_test.go:833: Expected deadline exceeded with valid auth, but have rpc error: code = Unknown desc = context deadline exceeded
FAIL
coverage: 23.2% of statements
FAIL	go.thethings.network/lorawan-stack/pkg/applicationserver	2.224s
```

#### Environment
<!--
Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs?
Paste the output of "ttn-lw-cli version" or "ttn-lw-stack version" if applicable.
-->

Travis

#### How do you propose to implement this?
<!-- Please think about how this could be fixed. -->

This is a weird error; the error code is `Unknown` but the error message is correct. Is this a leftover from a previous test or do we not return errors correctly?

#### Can you do this yourself and submit a Pull Request?
<!-- You can also @mention experts if you need help with this. -->

Can review and pray

--- a/pkg/applicationserver/applicationserver_test.go
+++ b/pkg/applicationserver/applicationserver_test.go
@@ -816,7 +816,7 @@ hardware_versions:
 				},
 			} {
 				t.Run(ctc.Name, func(t *testing.T) {
-					ctx, cancel := context.WithDeadline(ctx, time.Now().Add(Timeout))
+					ctx, cancel := context.WithDeadline(ctx, time.Now().Add(2*Timeout))
 					chs := &connChannels{
 						up:          make(chan *ttnpb.ApplicationUp, 1),
 						downPush:    make(chan *ttnpb.DownlinkQueueRequest),
@@ -855,7 +855,7 @@ hardware_versions:
 				}
 			}()
 			// Wait for connection to establish.
-			time.Sleep(Timeout)
+			time.Sleep(2 * Timeout)
 
 			t.Run("Upstream", func(t *testing.T) {
 				ns.reset()

